### ISSUE TITLE
New release for AGU?

### ISSUE THREAD

**Issue by rsignell-usgs** [intake/xrviz#65]
@hdsingh or @martindurant , could we get a new release before AGU next week?   Annie Burgess from ESIP will be including xrviz in her talk and it would be nice to have the latest functionality from master (which we are already using in the binder demo).   And there probably won't be much more development for a while now that @jsignell has moved to [SaturnCloud](https://www.saturncloud.io).  😢

**Comment by martindurant** [comment]
@hdsingh , can you handle this?

Oh, and I fully expect @jsignell to be working *flat out* on this project :)

**Comment by hdsingh** [comment]
Yes, sure. I made a [PyPi release](https://pypi.org/project/xrviz/0.1.4/#history). Conda-forge release would also be available shortly.

**Comment by martindurant** [comment]
Thanks @hdsingh !
